remediation/fence-agents-remediation/how-it-works.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: How It Works
+parent: Fence Agents Remediation
+grand_parent: Remediation
+nav_order: 1
+---
+
+## How Fence Agents Remediation works?
+
+Fence Agents Remediation (*FAR*) operator uses a management interface or a traditional API to remediate unhealthy kubernetes nodes.
```

Comment:
what's the difference between management interface and traditional API?

Reply:
For this context there is no much difference, and using `traditional API` or just ` API` should be enough to explain how the fence agent is communicating.